src/panels/config/helpers/forms/ha-input_select-form.ts

```diff
@@ -199,6 +265,8 @@ class HaInputSelectForm extends LitElement {
           border-radius: 4px;
           margin-top: 4px;
           --mdc-icon-button-size: 24px;
+          --mdc-ripple-color: transparent;
+          cursor: default;
```

Comment:
```suggestion
          --mdc-ripple-color: transparent;
          cursor: default;
          background-color: var(--card-background-color);
```
Can we add a background to have better styling when dragging?

**Current**
![CleanShot 2023-08-29 at 09 34 50](https://github.com/home-assistant/frontend/assets/5878303/6c1e5cd4-b3ac-4ade-ae39-126d7bdee2bb)

**Proposal**
![CleanShot 2023-08-29 at 09 35 31](https://github.com/home-assistant/frontend/assets/5878303/110f7c77-edef-44ed-bfe5-7b2ef2c8f9dd)